docs/source/glossary.md

```diff
@@ -17,4 +17,45 @@ Functional Programming
   This contrasts with functions or methods that depend on variables that are not explicitly passed as an input (such as accessing `self.variable` inside a method) or that alter the inputs or other state variables in-place, instead of returning new distinct variables as outputs.
 Dispatching
   Choosing which function or method implementation to use based on the type of the input variables (usually just the first variable). For some examples, see Python's documentation for the [singledispatch](https://docs.python.org/3/library/functools.html#functools.singledispatch) decorator.
+
 :::::
+
+```{glossary}
```

Comment:
Agree with sorting alphabetically!

Reply:
Right, noted. I guess I tried the backquotes as it was the format to be followed linked in the instructions (and because the initial glossary.md file with colon fence isn't rendering right on github (though it does on readthedocs, which is what's important)). Curious though, where does the colon fence format comes from (looked up [Add a Glossary](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/glossary.html) instructions but it's not there).

Reply:
The colon fence format comes from MyST. It supports both backtick and colon fences if configured: https://myst-parser.readthedocs.io/en/latest/syntax/optional.html#code-fences-using-colons. I prefer the colons for text heavy directives so even with a simple markdown renderer, most of the formatting looks good already. Specific myst things will still look as text and generally bad, but cursive, code, links... will already look good